src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/controller/ListingController.java

```diff
@@ -1,10 +1,17 @@
 package ch.uzh.ifi.fs22.sel.group15.cozycave.server.controller;
```

Comment:
please check for the new methods, that every method has proper debug messages. at the beginning something like .debug(is updat**ing**) and at the end .info(is updat**ed**)